app/clients/hedera/mirror-node/model/account/account.go

```diff
@@ -25,6 +25,41 @@ type (
 		Account string  `json:"account"`
 		Balance Balance `json:"balance"`
 	}
+
+	// AccountsQueryResponse struct used by the Hedera Mirror node REST API to return information regarding a given Account
+	AccountsQueryResponse struct {
+		Accounts []struct {
+			Account         string      `json:"account"`
+			Alias           string      `json:"alias"`
+			AutoRenewPeriod interface{} `json:"auto_renew_period"`
+			Balance         struct {
+				Timestamp string `json:"timestamp"`
+				Balance   int    `json:"balance"`
+				Tokens    []struct {
+					TokenId string `json:"token_id"`
+					Balance int    `json:"balance"`
+				} `json:"tokens"`
+			} `json:"balance"`
+			CreatedTimestamp              string      `json:"created_timestamp"`
+			DeclineReward                 bool        `json:"decline_reward"`
+			Deleted                       bool        `json:"deleted"`
+			EthereumNonce                 int         `json:"ethereum_nonce"`
+			EvmAddress                    string      `json:"evm_address"`
+			ExpiryTimestamp               interface{} `json:"expiry_timestamp"`
+			Key                           interface{} `json:"key"`
+			MaxAutomaticTokenAssociations int         `json:"max_automatic_token_associations"`
+			Memo                          string      `json:"memo"`
+			PendingReward                 int         `json:"pending_reward"`
+			ReceiverSigRequired           bool        `json:"receiver_sig_required"`
+			StakedAccountId               interface{} `json:"staked_account_id"`
+			StakedNodeId                  int         `json:"staked_node_id"`
+			StakePeriodStart              string      `json:"stake_period_start"`
+		} `json:"accounts"`
+		Links struct {
+			Next string `json:"next"`
+		} `json:"links"`
+	}
+
```

Comment:
I don't think we need to add all the info here, for now we just need the account numbers so leave only that